### PR TITLE
Fixes #34290 - Create pf4 breadcrumb switcher

### PR DESCRIPTION
**Pull request by MariaAga**
old:
Screenshot from 2022-01-18 19-55-48
new:
Screenshot from 2022-01-19 18-46-45

### CODE REVIEW

File: test/integration_test_helper.rb [hunk 1]
```diff
@@ -31,6 +31,7 @@
 Capybara.configure do |config|
   config.javascript_driver      = ENV["JS_TEST_DRIVER"]&.to_sym || :selenium_chrome
   config.default_max_wait_time  = 20
+  config.enable_aria_label = true
```
Comment (MariaAga) [inline test/integration_test_helper.rb:34]: @ezr-ondrej, can I just add things to ruby tests or do I need a new pr/discussion? :)
I need this to select things in the ruby test `test/integration/breadcrumbs_switcher_test.rb`
> **enable_aria_label** (Boolean = `false`) - Whether fields, links, and buttons will match against `aria-label` attribute.

Comment (ezr-ondrej) [reply test/integration_test_helper.rb:34]: I'd like this to be a separate commit, but just to appease my OCD. If it doesn't change or affect behaviour of other tests radically, you're fine to add simple config changes like this :+1:

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js [hunk 2]
```diff
@@ -69,33 +66,35 @@ class BreadcrumbBar extends React.Component {
           className="breadcrumbs-list"
         >
           {isSwitchable && (
-            <BreadcrumbSwitcher
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js:72]: This seems like we are not using this anymore, but we are still importing this old component, might be just forgotten import that got me confused ^_^

**Review by ezr-ondrej — COMMENTED** [review]
I do not feel entitled to review this as whole, but in general this looks quite ready to me.
Only thing I wonder here and am not able to figure out from the code: how do we deal with the currently used component? We just replace the Switcher for everyone (best case) or plugins still import some parts of the old component?

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js [hunk 3]
```diff
@@ -16,7 +17,7 @@ class BreadcrumbBar extends React.Component {
       resourceSwitcherItems,
     } = this.props;
     const isUrlFormatValid = resourceSwitcherItems.length
-      ? resourceSwitcherItems[0].url ===
+      ? resourceSwitcherItems[0].href ===
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js:20]: this is to fix a bug where the resources didn't update when moving from edit host to show host (https://github.com/theforeman/foreman/pull/6382)
Before 6382 - the resources was loaded one for edit hosts
after 6382 - the resources were loaded on every open as `resourceSwitcherItems[0].url` is undefined 
after changing to `href` - the resources are loaded only when the breadcrumbs are on a different page

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js (hunk 2 above)
Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js:72]: Thanks for the review! the import was a leftover and didn't get flagged as I used it for prop types, removed the old import and wrote the proptypes.
The switcher is part of the breadcrumbs, I don't think anyone (any plugin) uses it by itself and just use it through the breadcrumbs

**Review by ezr-ondrej — APPROVED** [review]
Works well, high-level the code LGTM, tho I do not feel like doing full React review, I'd be fine merging this nevertheless, unless @Ron-Lavi or @amirfefer have some blockers? :)

**Comment by ezr-ondrej** [comment]
Merging not to block this. We have 3 months to figure out if something is wrong, thanks @MariaAga :+1:

**Pull request merged**